lib/src/callback_debouncer.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+
+/// {@template callback_debouncer}
+/// A model used for debouncing callbacks.
+///
+/// Is only used internally and should not be used explicitly.
+/// {@endtemplate}
+class CallbackDebouncer {
```

Comment:
Since this file is not exported. No need to make this class visible for testing anymore.